collections/_technologies/digital-solutions-to-address-covid-19/healthcerts.html

```diff
@@ -20,161 +20,290 @@ <h2>HealthCerts</h2>
-    in Figure 1 below.
-    </p>
-    <figure style="text-align: center">
-      <img
-        src="/assets/img/usage-of-healthcerts.png"
-        alt="Confirming digital certificate legitmacy process"
-      />
-      <figcaption>Figure 1. Use of HealthCerts, Notarise and Verify</figcaption>
-    </figure>
-<h3>Digital Vaccine-Certificates in Singapore</h3>
-<p>
-  The Government Technology Agency(GovTech), in collaboration with the Singapore
-  Health Ministry have developed a system for issuing, authenticating and verifying
-  digital certificates shown in Figure 2 below. It ensures certificates issues are
-  tamper-proof, secure and comply with international standards and the Singapore
-  Government's requirements.
-  <figure style="text-align: center">
-    <img
-      src="/assets/img/verifying-digital-certificates.png"
-      alt="HealthCerts, Notarise and Verify"
-    />
-    <figcaption>Figure 2. System for issuing, authenticating and verifying digital
-    certificates</figcaption>
-  </figure>
-<h3>How It Works</h3>
-<p>
-  HealthCerts is a set of digital open standards and schema for issuing,
-  authenticating and verifying digital COVID-19 test results/vaccination certificates.
-  HealthCerts are issued in line with international standards
-  and the Singapore Government's requirements. It is interoperable with private
-  sector solutions. These certificates are also tamper-proof(data stored is immutable)
-  and secure, protected against unauthorised access.
-  </p>
-  <p>
-    The digital certificate stores identity details (name, national identity numnber, DOB etc.)
-    and test details (type, status, date, medical institution details). The QR
-    Code will be sent via email or presented under the SingPass Mobile app as shown
-    in Figure 3 below.
-    </p>
-    <figure style="text-align: center">
-      <img
-        src="/assets/img/HC on SingPass app.png"
-        alt="HealthCerts screenshot on SingPass mobile app"
-      />
-      <figcaption>Figure 3. HealthCerts presented on SingPass Mobile app</figcaption>
-    </figure>
-  <p>
-    HealthCerts is powered by <a href="https://www.openattestation.com/">OpenAttestation</a>,
-    an open-source framework using blockchain to issue cryptographically trustworthy
-    documents which can be verified independently without the need for proprietary
-    software equipment.
+    <strong>HealthCerts is designed to support an open ecosystem and
+      public-private partnerships. Singapore welcomes partners to further the
+      work in this space, including further enhancing interoperability, privacy,
+      and verifiability of digital certificates. Please contact us at info@tech.gov.sg should you have further queries. </strong>
+        <a href="https://www.youtube.com/watch?v=nRHgZ71gPmY">
+        <img alt="About HealthCerts" src="https://img.youtube.com/vi/https://www.youtube.com/watch?v=nRHgZ71gPmY/hqdefault.jpg">
```

Comment:
The video id is **nRHgZ71gPmY** .
We just need to replace the **id** instead of pasting the full links.
`http://img.youtube.com/vi/<insert-youtube-video-id-here>/mqdefault.jpg`

```suggestion
        <img alt="About HealthCerts" src="https://img.youtube.com/vi/nRHgZ71gPmY/hqdefault.jpg">
```